src/osha/oira/client/browser/client.py

```diff
@@ -20,20 +20,22 @@ def results(self):
         The format fits pat-autosuggest.
         There is a special label for the "all" list.
         """
-        # TODO: require query.
-        # q = self.request.get("q", "").strip().lower()
-        # if not q:
-        #     return []
+        q = self.request.get("q", "").strip().lower()
+        if not q:
+            return []
 
+        results = []
         catalog = api.portal.get_tool(name="portal_catalog")
-        all_users = {"id": "all", "text": "All OiRA users"}
-        # if q in all_users["id"] or q in all_users["text"].lower():
-
-        results = [all_users]
+        all_users = {"id": "general", "text": "All OiRA users"}
```

Comment:
I was thinking that this should be translated and probably it should be "All OiRA subscribers".
Not a blocker anyway

Reply:
Yes. Ideally the countries should also be translated, and queryable by their native names (or names in any language, maybe).

But I don't see the point in giving a mailing list a name that contains “subscribers”. That would make weird phrases like “subscribers to the "All OiRA subscribers" mailing list". Maybe it should be something like “General OiRA news” instead.

Reply:
I find it anyway better than "All OiRa users", but nevermind

Reply:
FTR in proto it seems to be “All users”, which imho is worse.

http://proto.quaivecloud.com/newsletters/special-offers#document-body-mailings

Reply:
If a non technical person like my brother reads a a term like "All OiRA subscribers", he understands the word "All" and does not understand the word "OiRA" and "subscribers". What is wrong with "All users"?

Reply:
It's a lie. Mails won't go to all users.

Reply:
Plus the other lists have names that refer to their topics (a country or tool name)

Reply:
FTR Alex says it needs to be called “All users” anyway

Reply:
Coming from the various calls with the oira partners and Lia where angel and me see how little they get the inner workings, I opt knowingly of the inconsistency for "All users", because it will give us least frictions with the oira partners/Lia and will result in them getting quicker to their goal of sending an email to a (even unsharply defined) target group.

Reply:
This does not address the point that mails will in fact not be sent to all users.

That said, I'm grateful for not having to discuss this any longer.